tasks/analysis.py

```diff
@@ -628,15 +628,27 @@ def process_impact_report(analysis, report_metadata):
         # TODO: find out how to upload document using post request
 
         assign_report = {
-            'map-report': analysis.assign_report_map,
+            # By default we always take the portrait report
+            'map-report-portrait': analysis.assign_report_map,
```

Comment:
I just realized that the default key is `inasafe-map-report-portrait`

Reply:
```suggestion
            'inasafe-map-report-portrait': analysis.assign_report_map,
```

Reply:
The strange thing is, it doesn't get caught in the unittest (or was it?). Maybe it was this line all along.